docker/Dockerfile

```diff
@@ -41,28 +41,36 @@ RUN ["dotnet", "build", "DfE.FindInformationAcademiesTrusts", "--no-restore", "-
 
 RUN ["dotnet", "publish", "DfE.FindInformationAcademiesTrusts", "--no-build", "-o", "/app"]
 
-### Install SQL tools to allow migrations to be run ###
-FROM "mcr.microsoft.com/dotnet/aspnet:${DOTNET_VERSION}-azurelinux3.0" AS base
-RUN curl "https://packages.microsoft.com/config/rhel/9/prod.repo" | tee /etc/yum.repos.d/mssql-release.repo
-ENV ACCEPT_EULA=Y
-RUN ["tdnf", "update"]
-RUN ["tdnf", "install", "-y", "mssql-tools18"]
-RUN ["tdnf", "clean", "all"]
+### Entity Framework: Migration Builder ###
+FROM build AS efbuilder
+WORKDIR /build
+ENV PATH=$PATH:/root/.dotnet/tools
+RUN ["mkdir", "/sql"]
+RUN ["dotnet", "tool", "install", "--global", "dotnet-ef"]
+RUN ["dotnet", "ef", "migrations", "bundle", "-r", "linux-x64", "-p", "DfE.FindInformationAcademiesTrusts", "--context", "DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Contexts.AcademiesDbContext", "--configuration", "Release", "--no-build", "-o", "/sql/migrateacademiesdb"]
+RUN ["dotnet", "ef", "migrations", "bundle", "-r", "linux-x64", "-p", "DfE.FindInformationAcademiesTrusts", "--context", "DfE.FindInformationAcademiesTrusts.Data.FiatDb.Contexts.FiatDbContext", "--configuration", "Release", "--no-build", "-o", "/sql/migratefiatdb"]
+COPY ./docker/init-docker-entrypoint.sh /sql/entrypoint.sh
+RUN ["chmod", "+x", "/sql/entrypoint.sh"]
+
+### Entity Framework: Migration Runner ###
+FROM "mcr.microsoft.com/dotnet/aspnet:${DOTNET_VERSION}-azurelinux3.0" AS initcontainer
+WORKDIR /sql
+COPY --from=efbuilder /sql /sql
+COPY --from=build /app/appsettings* /Dfe.FindInformationAcademiesTrusts/
```

Comment:
We can copy this from the repo rather than from the `build` stage which will remove the dependency between these stages